test/unit/cmap/connection_pool.test.js

```diff
@@ -438,7 +438,12 @@ describe('Connection Pool', function () {
     });
 
     loadSpecTests('connection-monitoring-and-pooling').forEach(test => {
-      it(test.description, function () {
+      const itFn =
+        test.description ===
+        'must aggressively timeout threads enqueued longer than waitQueueTimeoutMS'
```

Comment:
This test has been flaky for a long time, if we're okay with skipping I can make a ticket to take a look at this later.